tests/trx_generator/trx_provider.hpp

```diff
@@ -55,8 +58,28 @@ namespace eosio::testing {
          _connection_thread_pool.stop();
       };
 
+      fc::time_point get_trx_ack_time(const eosio::chain::transaction_id_type& _trx_id) {
+         fc::time_point time_acked;
+         _trx_ack_map_lock.lock();
```

Comment:
Use a `std::lock_guard g(_trx_ack_map_lock);`

Reply:
Thanks. Had been planning to do this, but completely forgot to come back and change it.
Addressed: [400c5b8](https://github.com/AntelopeIO/leap/pull/1165/commits/400c5b891a9f12e86916b246d1e4a073e39ec9e0)